### PR TITLE
modified file

### PR DESCRIPTION
**Pull request by GhosTRideR03**
Some things..

### CODE REVIEW

File: HW_Test/New_File.java [hunk 1]
```diff
@@ -2,5 +2,8 @@
 
 public class New_File {
 
+    public static void main(String[] args) {
+        int a = 0;
 
+    }
```
Comment (aliaskovski-idealo) [inline HW_Test/New_File.java:10]: =)

**Comment by ManolovGeorgiy** [comment]
**Cool**

File: HW_Test/New_File.java (hunk 1 above)
Comment (ManolovGeorgiy) [reply HW_Test/New_File.java:10]: (=

File: HW_Test/New_File.java [hunk 2]
```diff
@@ -2,5 +2,8 @@
 
 public class New_File {
```
Comment (aliaskovski-idealo) [inline HW_Test/New_File.java:4]: add SystemOutPrint

**Comment by GhosTRideR03** [comment]
Done

**Pull request merged**